tutorial/whatsup/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/prometheus/client_golang/tutorial
 
-go 1.20
+go 1.21
```

Comment:
I believe we want to keep 1.20 until the next go release

Reply:
Oops. Yes, absolutely. Just a leftover from when I reverted the changes I had made before `common` was fixed in this sense.